examples/spark/src/main/kotlin/com/expediagroup/graphql/examples/spark/Application.kt

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright $today.year Expedia, Inc
```

Comment:
All of these need to have `$today.year` replaced. If you are using an IDE like IntelliJ you can set this text as the template otherwise you will have to manually add this to all files

![Screen Shot 2020-03-30 at 12 44 37 PM](https://user-images.githubusercontent.com/2446877/77954958-4bbabc00-7284-11ea-95a3-096a0b28aab5.png)

Reply:
lol sorry

Reply:
Date set